src/examples/src/org/apache/poi/xssf/usermodel/examples/BarAndLineChart.java

```diff
@@ -134,17 +134,26 @@ public static void main(String[] args) throws Exception {
             lineCategories.crossAxis(rightValues);
 
             // the bar chart
-            XDDFBarChartData bar = (XDDFBarChartData) chart.createData(ChartTypes.BAR, lineCategories, rightValues);
+            XDDFBarChartData bar = (XDDFBarChartData) chart.createData(ChartTypes.BAR, barCategories, leftValues);
             XDDFBarChartData.Series series1 = (XDDFBarChartData.Series) bar.addSeries(xs, ys1);
             series1.setTitle("Bars", new CellReference("Sheet1!$B$1"));
             bar.setVaryColors(true);
             bar.setBarDirection(BarDirection.COL);
             chart.plot(bar);
 
-            // the line chart
+            // the line chart on secondary axis
             XDDFLineChartData lines = (XDDFLineChartData) chart.createData(ChartTypes.LINE, lineCategories,
                     rightValues);
+            
+            //uncomment below line if only primary axis required and comment above line 
+            // the line chart on primary axis
+            /*XDDFLineChartData lines = (XDDFLineChartData) chart.createData(ChartTypes.LINE, lineCategories,
+                    leftValues);*/
+            
+            
             XDDFLineChartData.Series series2 = (XDDFLineChartData.Series) lines.addSeries(xs, ys2);
+            series2.updateIdXVal(1);
+            series2.updateOrderVal(1);
```

Comment:
so we created update methods, because in a chart, we need to provide unique value to each series, now while creating combo chart, i.e. line series and bar series in that case we assigning same values to each series due to that it was corrupting the office file. Now either we can create a new constructor with new parameter i.e. index or can provide an update method to re-assign axis id and order id. so i choosed update method.